dart/lib/flat_buffers.dart

```diff
@@ -126,15 +129,15 @@ class Builder {
 
   /// The location of the end of the current table, measured in bytes from the
   /// end of [_buf], or `null` if a table is not currently being built.
-  int _currentTableEndTail;
+  int _currentTableEndTail = 0;
```

Comment:
Is zero the right sentinel value to use?

Doc comment needs to be updated, since it can't be null anymore.

Reply:
it is never accessed before writing, setting a value just makes it a bit faster internally because dart doesn't have to consider it ever being `null`. Alternatively, I can change it to `late` though I guess that comes with some runtime overhead.